tasks/list.yml

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: MIT
+---
+- name: List snapshots
+  ansible.builtin.script: "{{ __snapshot_cmd }}"
+  args:
+    executable: "{{ __snapshot_python }}"
+    body_format: json
```

Comment:
where is `body_format` documented?

Reply:
It is a mistake will fix.

It is only valid for the uri_module: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/uri_module.html